webpack.config.js

```diff
@@ -52,11 +53,27 @@ const font_loader = {
     },
 }
 
+//Require returns raw content of files.
+const raw_loader = {
+    test: /\.(cpp)$/,
+    use: 'raw-loader'
+}
+
+const asciidoc_loader = {
+    test: /\.(adoc)$/,
+    use: [
+        'html-loader',
+        'asciidoctor-loader?attributes=showtitle'
+    ]
+}
+
 const js_loader = {
     test: /\.jsx?$/,
+    exclude: /(node_modules|bower_components)/,
     use: [
         "babel-loader",
-        "eslint-loader"
+        //Disabled temporarely due to erros from its dependency objest-hash
```

Comment:
Can we still run eslint as part of `npm start`, but not as part of refreshing inside webpack?

Reply:
I'll check out